firewood/src/merkle.rs

```diff
@@ -1175,6 +1178,220 @@ impl<S: ShaleStore<Node> + Send + Sync> Merkle<S> {
     pub fn flush_dirty(&self) -> Option<()> {
         self.store.flush_dirty()
     }
+
+    pub fn get_iter<K: AsRef<[u8]>>(
+        &self,
+        key: Option<K>,
+        root: DiskAddress,
+    ) -> Result<MerkleKeyValueStream<'_, S>, MerkleError> {
+        // TODO: if DiskAddress::is_null() ...
```

Comment:
Will all this TODOs be tracked in a issue so that we won't forget about it? All of them seem a must do.

Reply:
This was fixed in #346